src/ORM/DatabaseAdmin.php

```diff
@@ -190,9 +207,18 @@ public function buildDefaults()
      *
      * @return string Returns the timestamp of the time that the database was
      *                last built
+     *
+     * @deprecated 5.4.0 Will be replaced with SilverStripe\Dev\Command\DbBuild::lastBuilt()
      */
     public static function lastBuilt()
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice(
+                '5.4.0',
+                'Will be replaced with SilverStripe\Dev\Command\DbBuild::lastBuilt()'
+            );
+        });
+
```

Comment:
```suggestion
     */
    public static function lastBuilt()
    {
```
Class is already deprecated and method name will be the same

Reply:
This is a static method. It's possible to call this method without instantiating the class, which means we need a deprecation warning in the method itself.